gtk/FaviconCache.cc

```diff
@@ -97,7 +97,7 @@ bool favicon_web_done_idle_cb(std::unique_ptr<favicon_data> fav)
     }
 
     // Not released into the next web request, means we're done trying (even if `pixbuf` is still invalid)
-    if (fav != nullptr)
+    if (fav)
```

Comment:
I think we could agree that no deference was actually happening here, and I also thought that we were trying to make conditions explicitly boolean (given that `fav` is a pointer, even if a smart one). This looks more like a style change than an issue fix...

What Coverity warns about is that there's `fav` dereference on line 85 above without a prior null check, so I'm not sure this fix will even help. Maybe we should have a separate flag to indicate that `fav` is no longer valid after release on line 96, or maybe we could just `return false` there right away so this check here isn't necessary.